packages/cli/lib/lib/webpack/webpack-client-config.js

```diff
@@ -278,6 +278,43 @@ function isProd(env) {
 	return prodConfig;
 }
 
+function setupProxy(target) {
+	if (!target) {
+		return;
+	}
+
+	const errorTemplate = warn =>
+		`Invalid proxy configuration in package.json. ${warn} Skipping and using default.`;
+
+	if (typeof target !== 'string') {
+		warn(errorTemplate('If provided, "proxy" must be a string.'));
+		return;
+	} else if (!/https?:\/\//.test(target)) {
+		warn(
+			errorTemplate(
+				'If provided, "proxy" must start with "http://" or "https://".'
+			)
+		);
+		return;
+	}
+
+	return {
+		target,
+		logLevel: 'warn',
```

Comment:
This will result in proxy issues appearing in the user's terminal like the following:

![temp](https://user-images.githubusercontent.com/33403762/158286040-4186d878-e8c7-4080-804d-8252e16d6e4f.png)

I tend to think that's handy, but we certainly could disable it and leave those errors to the browser console too.